### ISSUE TITLE
Request support for guzzlehttp/guzzle @ ^7.0.1

### ISSUE THREAD

**Issue by mikerodham** [wondeltd/php-client#13]
Started to upgrade our application today in which we need to satisfy guzzle being at least `^7.0.1`, which is not supported by this package. Do you have any plan to support this?

**Console Output**
```
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for guzzlehttp/guzzle 7.0.1 -> satisfiable by guzzlehttp/guzzle[7.0.1].
    - wondeltd/php-client 2.1.0 requires guzzlehttp/guzzle ^6.0.0 -> satisfiable by guzzlehttp/guzzle[6.5.x-dev].
    - Can only install one of: guzzlehttp/guzzle[7.0.1, 6.5.x-dev].
    - Installation request for wondeltd/php-client 2.1.0 -> satisfiable by wondeltd/php-client[2.1.0].
```

**Comment by benlumley** [comment]
My fork is working with Guzzle 7;  still in process of testing it doesn't break backwards compat - so haven't PR'd

**Comment by RobMethven** [comment]
@benlumley See you've PRd now, so assume it's good to go..?

@wondedeployer Please accept Ben's PR!

**Comment by benlumley** [comment]
I've got it in production for a few months.

You should be able to pull it from my branch until wonde merge the PR, just add this in to the repositories section of composer.json and then change your version requirement to dev-master....

````
"repositories": [
        {
            "type": "vcs",
            "url": "https://github.com/benlumley/php-client"
        },
       
    ],
````

**Comment by RobMethven** [comment]
Thanks Ben - managed to do that not long after I commented. Nice work!

**Comment by mikerodham** [comment]
Appreciate this guys, great work. Our application has been fully updated and all dependencies satisfied. Thank you!

**Comment by RobMethven** [comment]
> Appreciate this guys, great work. Our application has been fully updated and all dependencies satisfied. Thank you!

Can't seem to see any changes, and if I try to add using composer I get the same error about Guzzle 6.

**Comment by benlumley** [comment]
You need:

````
    "repositories": [
        {
            "type": "vcs",
            "url": "https://github.com/benlumley/php-client"
        }
    ],
````

and then change the require to:
````
      "wondeltd/php-client": "dev-master"
````

**Comment by RobMethven** [comment]
> You need:
> 
> ```
>     "repositories": [
>         {
>             "type": "vcs",
>             "url": "https://github.com/benlumley/php-client"
>         }
>     ],
> ```
> 
> and then change the require to:
> 
> ```
>       "wondeltd/php-client": "dev-master"
> ```

Sorry Ben, yes I know I can do this. Just didn't understand why the issue seemed to have been closed without merging those changes.

**Comment by benlumley** [comment]
Unsure - wasn't my issue; the original guy closed it - I've got #17 open to merge this in.

**Comment by RobMethven** [comment]
Ahhh, my fault for not paying attention.
